query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "      <td>11216</td>\n",
+       "      <td>11149</td>\n",
+       "      <td>11207</td>\n",
+       "      <td>http://loinc.org</td>\n",
+       "      <td>34117-2</td>\n",
+       "      <td>None</td>\n",
+       "      <td>final</td>\n",
+       "      <td>4285</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>11216</td>\n",
+       "      <td>11149</td>\n",
+       "      <td>11207</td>\n",
+       "      <td>http://loinc.org</td>\n",
+       "      <td>51847-2</td>\n",
+       "      <td>None</td>\n",
+       "      <td>final</td>\n",
+       "      <td>4285</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>11633</td>\n",
+       "      <td>10669</td>\n",
+       "      <td>11625</td>\n",
+       "      <td>http://loinc.org</td>\n",
+       "      <td>34117-2</td>\n",
+       "      <td>None</td>\n",
+       "      <td>final</td>\n",
+       "      <td>4409</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>11633</td>\n",
+       "      <td>10669</td>\n",
+       "      <td>11625</td>\n",
+       "      <td>http://loinc.org</td>\n",
+       "      <td>51847-2</td>\n",
+       "      <td>None</td>\n",
+       "      <td>final</td>\n",
+       "      <td>4409</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>12336</td>\n",
+       "      <td>11149</td>\n",
+       "      <td>12334</td>\n",
+       "      <td>http://loinc.org</td>\n",
+       "      <td>34117-2</td>\n",
+       "      <td>None</td>\n",
+       "      <td>final</td>\n",
+       "      <td>4285</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   dr_id patient_id encounter_id            system     code result_obs_id  \\\n",
+       "0  11216      11149        11207  http://loinc.org  34117-2          None   \n",
+       "1  11216      11149        11207  http://loinc.org  51847-2          None   \n",
+       "2  11633      10669        11625  http://loinc.org  34117-2          None   \n",
+       "3  11633      10669        11625  http://loinc.org  51847-2          None   \n",
+       "4  12336      11149        12334  http://loinc.org  34117-2          None   \n",
+       "\n",
+       "  status practitioner_id  \n",
+       "0  final            4285  \n",
+       "1  final            4285  \n",
+       "2  final            4409  \n",
+       "3  final            4409  \n",
+       "4  final            4285  "
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_diagnostic_report AS\n",
+    "      SELECT D.id AS dr_id, D.subject.patientId AS patient_id,\n",
```

Comment:
For DiagnosticReport, suggest adding:

* category
* conclusionCode
* conclusion